src/lara-plugin/plugin-api/sidebar.ts

```diff
@@ -1,7 +1,10 @@
+// LARA_CODE: import * as $ from "jquery";
 // ACTIVITY_PLAYER_CODE:
 import $ from "jquery";
-// LARA_CODE: import * as $ from "jquery";
-// LARA_CODE: import "jquery-ui/ui/widgets/button";
+(window as any).jQuery = $;
+(window as any).$ = $;
+
+import "jquery-ui/ui/widgets/button";
```

Comment:
Given that an initialization sequence like this occurs in multiple places, I wonder if it would make sense to put it all in a single file that can be imported elsewhere, e.g. `import-jquery.ts` or `init-jquery.ts`.

Reply:
I'm going to leave it inline for now - I think it makes it a little easier to diff and compare against the original (which was Scott's intention with this work).  But I'll be open to this if I have to continue with the import pattern.